benchmarks/README.md

```diff
@@ -62,3 +62,17 @@ is an example of a cell output that takes a while to render:
 ```bash
 env 'BENCHMARK_NOTEBOOKS=["./fixedDataTable", "./manyOutputs"]' jlpm all
 ```
+
+## Comparing Benchmarks
+
+We also support comparing two runs of benchmarks to give a sense of the relative changes:
+
+```bash
+# Create `old.csv` and `new.csv` files
+env 'BENCHMARK_NOTEBOOKS=["./manyOutputs", "./longOutput"]' BENCHMARK_OUTPUT=old.csv jlpm start:benchmark
+env 'BENCHMARK_NOTEBOOKS=["./manyOutputs", "./longOutput"]' BENCHMARK_OUTPUT=new.csv jlpm start:benchmark
+# Create comparison csv file
+jlpm run start:compare
+# Create and open graph to view results
+jlpm run start:compare-graph
+```
```

Comment:
You link to the paper in the source comment. Would be good to also mention that in the README.

I have started reading the paper to understand how to interpret the produced graphs but I feel we need a summary also in this README to know how to quickly interpret them.

In the description of this PR, you say  it will say `"The new run is X%+-Y faster than the old"`.  I see the script prints something, but when we will run that via the CI, we may want to have a file written (json?) so it can be reused by a bot.


```
Quantifies the performance changes between two measures systems. Assumes we gathered
 * n independent measurement from each, and calculated their means and varience.
 *
 * Based on the work by Tomas Kalibera and Richard Jones. See their paper
 * "Quantifying Performance Changes with Effect Size Confidence Intervals", section 6.2,
 * formula "Quantifying Performance Change".
 *
 * However, it simplifies it to only assume one level of benchmarks, not multiple levels.
 * If you do have multiple levels, simply use the mean of the lower levels as your data,
 * like they do in the paper.
```